### PR TITLE
Cures #3064 for 3.2.3

### PR DESCRIPTION
**Pull request by AHaumer**
This is a bug fix for #3064. 
Maybe the problem occurs rather seldom, bu should got to the next maintenance release.

### CODE REVIEW

File: Modelica/Electrical/Analog/Basic.mo [hunk 1]
```diff
@@ -311,15 +314,14 @@ package Basic "Basic electrical components"
...
-      Modelica.Math.atan(Ipar/Inom));
+    (Lnom - Linf)/(Lzer - Linf)=Ipar/Inom*(pi/2 - atan(Ipar/Inom));
   equation
     assert(Lzer > Lnom + Modelica.Constants.eps, "Lzer (= " + String(Lzer) +
       ") has to be > Lnom (= " + String(Lnom) + ")");
     assert(Linf < Lnom - Modelica.Constants.eps, "Linf (= " + String(Linf) +
       ") has to be < Lnom (= " + String(Lnom) + ")");
-    (Lact - Linf)*i/Ipar = (Lzer - Linf)*noEvent(Modelica.Math.atan(i/Ipar));
-    Psi = Lact*i;
+    Lact = Linf + (Lzer - Linf)*(if noEvent(abs(i)/Ipar<small) then 1 else atan(i/Ipar)/(i/Ipar));
```
Comment (beutlich) [inline Modelica/Electrical/Analog/Basic.mo]: I'd still prefer Modelica.Math.atan here (see the discussion in #2090).

**Comment by beutlich** [comment]
To avoid double reviews, it would be better to only open one PR for the master branch. If it is labeled as bug, I will do the cherry-picking once it is merged and create the follow-up PR for maint/3.2.3 afterwards.

**Comment by AHaumer** [comment]
@beutlich : Sorry for trying to avoid cherry-picks...
Fixed eps and atan. 
Though Simulation-X is smart enough to master a critical situation, the implemenation should be clean.

**Comment by AHaumer** [comment]
@beutlich Should I close this PR if you prefer cherry picking from #3066 ?

**Comment by beutlich** [comment]
> 
> 
> @beutlich Should I close this PR if you prefer cherry picking from #3066 ?

No, we can leave it open - just ignore it for now. I will force-push the cherry-picked changes from master to this branch later.

**Pull request merged**